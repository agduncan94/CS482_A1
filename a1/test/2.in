ATCTATGTCTGATAGAGGGAGACCTTATATAAATATATATATA
ACTGATTG
>header seq1
ACTC
