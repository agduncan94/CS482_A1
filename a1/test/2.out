AAA 0.0196
AAC 0.0196
AAG 0.0000
AAT 0.0196
ACA 0.0000
ACC 0.0196
ACG 0.0000
ACT 0.0392
AGA 0.0392
AGC 0.0000
AGG 0.0196
AGT 0.0000
ATA 0.1569
ATC 0.0196
ATG 0.0196
ATT 0.0196
CAA 0.0000
CAC 0.0000
CAG 0.0000
CAT 0.0000
CCA 0.0000
CCC 0.0000
CCG 0.0000
CCT 0.0196
CGA 0.0000
CGC 0.0000
CGG 0.0000
CGT 0.0000
CTA 0.0196
CTC 0.0196
CTG 0.0392
CTT 0.0196
GAA 0.0000
GAC 0.0196
GAG 0.0392
GAT 0.0392
GCA 0.0000
GCC 0.0000
GCG 0.0000
GCT 0.0000
GGA 0.0196
GGC 0.0000
GGG 0.0196
GGT 0.0000
GTA 0.0000
GTC 0.0196
GTG 0.0000
GTT 0.0000
TAA 0.0392
TAC 0.0000
TAG 0.0196
TAT 0.1373
TCA 0.0000
TCC 0.0000
TCG 0.0000
TCT 0.0392
TGA 0.0392
TGC 0.0000
TGG 0.0000
TGT 0.0196
TTA 0.0196
TTC 0.0000
TTG 0.0196
TTT 0.0000
